### ISSUE TITLE
Fibaro smoke detector (FGSD-002) not initialized

### ISSUE THREAD

**Issue by martinjo** [cdjackson/HABmin#288]
I'm using the latest stable version of Openhab with Habmin (downloaded master.zip from github) together with a Aeon Z-wave gen 5 stick and a Fibaro FGSD-002 smoke detector.

The smoke detector always stays grey in the Habmin interface. I've managed at some point to see the configuration parameters from the Product database for the device, but then they disappeared again, and I don't understand why.

I paired my detector using the button on the z-wave stick since I'm not sure the include/exclude buttons in Habmin are working (I get no feedback of device inclusion in the log).

Any ideas of how to include the Fibaro device?

**Comment by cdjackson** [comment]
When including battery devices, you will need to wake the device up manually - this may take a few attempts as there is a lot of traffic that needs to be exchanged....

So, including using the button on the stick is fine. After plugging back in, you will need to wake the device up (normally by pressing the button a few times). This might need to be repeated a few times - possibly waiting 30 seconds between each attempt. This is to allow all the data to be exchanged during the device discovery phase.

I would need debug a log to work out what's happening though.

**Comment by cdjackson** [comment]
I think this really should be discussed on the forum, and not here as others will also be able to assist...

**Comment by martinjo** [comment]
Thanks! 

I know got the device to work. I pressed Reinitialise and then woke the device up numerous times (by pressing the button three times). After a few tries it got the configuration parameters. And then after a while the grey dot turned yellow. Now the dot is back to grey, but the device is working. I even tested the actual smoke alarm by lighting two matches and I got the "Open" signal as expected.

The initialization sequence seems really strange to me as there is no real feedback to what is going on and how many times I need to wake my device. But now that it's working I'm happy :-)
